images/nodesvc-base/Dockerfile

```diff
@@ -1,15 +1,20 @@
-FROM registry.scality.com/federation/nodesvc-base:7.10.5.0
+FROM registry.scality.com/federation/nodesvc-base:7.10.7.0 as builder
 
-WORKDIR ${HOME_DIR}/backbeat
+WORKDIR /tmp/build
 
-COPY ./package.json ./yarn.lock ${HOME_DIR}/backbeat
+COPY ./package.json ./yarn.lock /tmp/build
 
-# Remove when gitcache is sorted out
-RUN rm /root/.gitconfig
+RUN yarn cache clean \
+    && yarn install --production --frozen-lockfile --ignore-optional --ignore-engines --network-concurrency 1 \
+    && yarn cache clean
 
-RUN yarn install --production --frozen-lockfile --network-concurrency 1
+################################################################################
+FROM registry.scality.com/federation/nodesvc-base:7.10.5.0
```

Comment:
maybe a good time to bump the base image to `7.10.7` as well?

Reply:
my mistake. had only bumped the builder base